tests/test_config.py

```diff
@@ -679,6 +681,20 @@ def test_get_project_id(self):
 
     self.assertEqual(self.config_dict['projectId'], self.project_config.get_project_id())
 
+  def test_get_bot_filtering(self):
+    """ Test that bot filtering is retrieved correctly when using get_bot_filtering_value. """
+
+    # Assert bot filtering is False when not provided in data file
+    self.assertTrue('botFiltering' not in self.config_dict)
+    self.assertEqual(False, self.project_config.get_bot_filtering_value())
```

Comment:
I need to check, when botFiltering is not in dictionary what it's supposed to return? it MUST be NULL, i haven't seen such condition in JS.